front/app/components/admin/Graphs/BarChart/index.tsx

```diff
@@ -1,165 +1,29 @@
-import React, { RefObject } from 'react';
-import { isEmpty } from 'lodash-es';
+import React from 'react';
 
 // components
-import {
-  ResponsiveContainer,
-  BarChart as RechartsBarChart,
-  Bar,
-  XAxis,
-  YAxis,
-  Cell,
-} from 'recharts';
-import { NoDataContainer } from 'components/admin/GraphWrappers';
-
-// hooks
-import { useTheme } from 'styled-components';
-
-// i18n
-import messages from '../messages';
-import { FormattedMessage } from 'utils/cl-intl';
+import MultiBarChart, {
+  Props as MultiBarChartProps,
```

Comment:
Why not directly name this type MultiBarChartProps instead of having to rename it on every import?

Reply:
Usually when we make a single-file component we just call the props `Props`. Renaming them to `MultiBarChartProps` makes the importing a bit easier but from the `MultiBarChart/index` file it gives me the impression that you are defining multiple components/props in one file